### ISSUE TITLE
Planning Entity Page Layouts: Improving SR

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#345]
The list page content was changed to provide the best experience for a SR (screen reader) user. What was changed was limiting the number of elements per card/entity that the SR would navigate through, while still providing access to all functionality that a visual user would have access to. Entity pages now need the same rework, but as it is not a list page where a user would want to go through many cards and only require limited information, this is an entities full page. That means the user will need individual access to specific information and functionality, instead of a quick overview summary of information. This will require a different approach.

**Task**: Decide the general default layout of information and functionalities to be accessible on entity pages. 
**Key Point**: Screen Readers by default will read each element one at a time, need to make their experience smooth. 

What information and functionality is important to provide? How should it be provided? What functionality from the MoreOptions menu should be moved directly into the page? 
What information should be added that is not already provided, for visual users as well?

Reference: List Pages, Twitter: List vs Tweet,

**Comment by bmrgould** [comment]
Situations where a component opens a component:

1. MoreOptions -> UserSheet
2. MoreOptions -> Report (Dialog)
3. UserSheet -> Send Message (Dialog)

One is solved by the PR.
Two and Three should be solved by having dedicated pages for Reporting and New Messages.

**Comment by bmrgould** [comment]
Place Holder

**Comment by bmrgould** [comment]
Action Bar: Placeholder
